scss/standalone/patterns_segmented-control.scss

```diff
@@ -0,0 +1,6 @@
+@import '../vanilla';
+@include vf-base;
+
+@include vf-p-segmented-control;
+@include vf-p-buttons;
```

Comment:
Does it need to rely on `p-buttons`? New components should be independent from each other and only rely on base styles ideally.

Reply:
This is a bit of a gray area IMO.  This segmented control is a collection of buttons. What would be the alternative - move most of the button code into base?

Reply:
We looked at it as we were just adding a wrapper around buttons, but dropping their border styles - I think it'd be best to chat it out after standup.